src/networkd.c

```diff
@@ -27,6 +27,8 @@
 #include "parse.h"
 #include "util.h"
 
+/* netplan-feature: renderer_networkd */
```

Comment:
I would recommend not exposing flags for features that have been present "forever" as this just makes the output more verbose without providing value.

There will never be a version of netplan that both supports the 'netplan info' command, and does not have support for a networkd renderer... unless you intend to make this future-proof in case later versions of netplan *drop* support for renderers?

Either way, I think any of these features that are present in the schema in 0.36.1 (the version in bionic release) don't need to be declared.

Reply:
Would it make sense to have feature flags for "forever" features that might not necessarily be available?  I'm thinking specifically of network manager; would it make sense to only provide a network-manager renderer feature flag if network manager is actually installed?

Reply:
It certainly makes sense to me to emit which *renderers* the current netplan binary supports.  Cloud-init currently assumes that installing netplan implies that the target system which has netplan can support *all* renderers that netplan does.  Looking forward, they may not be reasonable unless we want netplan itself to depend on any renderer it supports configuring.  If a renderer backed were optional, then I think cloud-init would be interested in knowing if the target netplan would accept a particular yaml with backend specific bits.